### ISSUE TITLE
How to pass command line arguments to running application?

### ISSUE THREAD

**Issue by nartamonov** [kybu/headless-selenium-for-win#7]
I was unable to find out proper way to pass command line arguments to application that I want to run within virtual desktop with the help of desktop_utils. For example, I want to open some URL in Firefox instance. Following command is not working:

```
C:\Program Files (x86)\Mozilla Firefox>desktop_utils --desktop D1 --run "firefox.exe http://masterhost.ru/"
Could not create the 'firefox.exe http://masterhost.ru/' process in the 'D1' desktop!
```

**Comment by kybu** [comment]
I will check that.

**Comment by kybu** [comment]
Sorry, that it took a while. It is implemented in version 1.3.
